src/components/NcActionRadio/NcActionRadio.vue

```diff
@@ -116,14 +141,28 @@ export default {
 			return !this.disabled
 		},
 
+		model: {
+			get() {
+				return this.modelValue
+			},
+			set(value) {
+				/**
+				 * Emitted when the radio state is changed
+				 *
+				 * @type {string|number}
+				 */
+				this.$emit('update:modelValue', value)
+			},
+		},
```

Comment:
In Vue 3 it can be replaced with built-in `useModel`
https://vuejs.org/api/composition-api-helpers.html#usemodel